### ISSUE TITLE
Hi，能否详细讲一下论文中是如何划分训练集，测试集和交叉验证的

### ISSUE THREAD

**Issue by DinghaoXi** [Minerva-jiezhao/PGU-net-Model#2]

**Comment by Minerva-J** [comment]
> _No description provided._

Because there are different cells in the data set, we randomly select and divide the training set, verification set and test set
